src/runtime_src/ert/ert.mk

```diff
@@ -1,5 +1,4 @@
-RDI_DEVKITS_HOME := /tools/batonroot/rodin/devkits
-MB_HOME := $(RDI_DEVKITS_HOME)/lnx64/cross/RHEL6/microblaze/lin
+MB_HOME := ${XILINX_VITIS}/gnu/microblaze/lin
```

Comment:
Are we sure this is the same compiler?  This is a pretty scary change unless we are certain the compiler to use.

Reply:
Even if I set XILINX_VITIS and use the MB_HOME, the sched.bin file are not identical to before.

While I agree with the intent of this PR and that using XILINX_VITIS is better that using the internal mb compiler,  making this change in 2019.2 branch is not possible.      

Please redo this PR and make it specific to what you are after, such that it does not affect building of the microblaze firmware used in hardware.